PixivConfig.py

```diff
@@ -60,6 +60,7 @@ class PixivConfig():
         ConfigItem("Network", "retryWait", 5),
         ConfigItem("Network", "downloadDelay", 2),
         ConfigItem("Network", "checkNewVersion", True),
+        ConfigItem("Network", "openNewVersion", False),
```

Comment:
change the default to True, else no one going to be aware of this feature.

If they don't want to use it, they can turn it off by themself